.github/workflows/deploy.yml

```diff
@@ -195,5 +197,47 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-      - name: TODO - Documentation Deploy Steps
-        run: echo "TODO - Documentation Deploy Steps"
+      - name: Documentation Deploy Steps
+        run: echo "Documentation Deploy Steps"
+
+      - name: Checkout Project
+        uses: actions/checkout@v3
+
+      - name: Setup Node
+        uses: actions/setup-node@v3.7.0
+        with:
+          node-version-file: '.nvmrc'
+          registry-url: 'https://registry.npmjs.org'
+          cache: 'yarn'
+
+      - name: Uncache Dependencies
+        uses: actions/cache@v3
+        with:
+          path: '**/node_modules'
+          key: node-modules-${{ hashFiles('./yarn.lock') }}
+
+      - name: Uncache Distributables
+        uses: actions/cache@v3
+        with:
+          path: '**/dist'
+          key: dist-${{ env.rid }}
+
+      - name: Synchronize Packages
+        run: yarn
+
+      - name: Build Docs
+        run: yarn run build:docs
+
+      - name: Build Samples
+        run: yarn run samples:build
+
+      - name: Set Git Identity
+        run: |
+          git config user.email "${GIT_AUTHOR_EMAIL}"
+          git config user.name "${GIT_AUTHOR_NAME}"
+
+      - name: Publish Docs Folder
+        run: |
+          git add docs/
+          git commit -m "chore(docs): update docs"
```

Comment:
i would add the versoin number if possible, Not sure if we want to do it here but only if webex package  changes then only we need to deploy the documentation we could optimise it later too

Reply:
We are not publishing the Webex package yet so we will add this later